src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -70,7 +70,7 @@ import Disclaimer from '../../components/Disclaimer';
 import SecondaryColumn from './SecondaryColumn';
 import styles from './ArticlePage.styles';
 import { ComponentToRenderProps, TimeStampProps } from './types';
-import ContinueReadingButton from './ContinueReadingButton';
+import ContinueReadingButton, { Props as ContineReadingProps } from './ContinueReadingButton';
```

Comment:
Can we fix this typo?
```suggestion
import ContinueReadingButton, { Props as ContinueReadingProps } from './ContinueReadingButton';
```